### PR TITLE
[onert] Optimize StridedSlice

### PR DESCRIPTION
**Pull request by YiHyunJin**
when only last dimension used, we have easy accesse to index.
So I optimized it for that situation.
And tested any models. The result is an 3.5% increase in speed.
(1.233 -> 1.191)

Signed-off-by: YiHyunjin <hj0412.yi@samsung.com>

### CODE REVIEW

File: compute/cker/include/cker/operation/StridedSlice.h [hunk 1]
```diff
@@ -260,12 +260,38 @@ template <typename T>
...
+  bool optimize = true;
+  int st_count = op_params.strides_count;
+  for (int idx = 0; idx < st_count - 1; idx++)
+  {
+    if (op_params.start_indices[idx] != 0 || op_params.stop_indices[idx] != 0)
+    {
+      optimize = false;
+      break;
+    }
+  }
```
Comment (glistening) [inline compute/cker/include/cker/operation/StridedSlice.h:275]: @YiHyunJin Why don't you check in KernelGenerator?

**Comment by wateret** [comment]
> @YiHyunJin Why don't you check in KernelGenerator?

How was this resolved? It looks like the param check is done in the kernel. Just adding some more description, for example,

- Split the optimized kernel and original kernel into two separate functions
- As we know the params at prepare(compile) time, move the param check code in KernelGenerator
- Then we are able to bind the most proper kernel for that single operation node, and no need to check it at execution time

BTW it looks good itself and this work can be done after merging this PR.

**Review by wateret — CHANGES_REQUESTED** [review]
Please add a test case (for code coverage and verification)

**Comment by wateret** [comment]
@nnfw-bot test android-runtime-builder

**Comment by glistening** [comment]
`android-runtime-builder` fails because downloading 'http://sourceforge.net/projects/boost/files/boost/1.58.0/boost_1_58_0.tar.gz' failed.

**Comment by glistening** [comment]
I've write testcase for this PR using CircleGen. Please see #4525.

**Comment by wateret** [comment]
@YiHyunJin Could you please respond https://github.com/Samsung/ONE/pull/4508#issuecomment-704742171 ?

**Comment by YiHyunJin** [comment]
@wateret It looks good to do after `buildStridedSliceParams`.
So I checked in `StridedSlice`.

> BTW it looks good itself and this work can be done after merging this PR.
Yes~

**Review by glistening — APPROVED** [review]
LGTM

**Review by wateret — APPROVED** [review]
LGTM

**Comment by glistening** [comment]
@YiHyunJin This morning, I've tested with merged PR. In my environment (a bit different model), this PR does not make differences for our target model.

1.010 → 1.005 (about 0.49% improvement)

**Comment by YiHyunJin** [comment]
@glistening  I checked this problem and i fixed.
https://github.com/Samsung/ONE/pull/4543

**Pull request merged**